documentation/release_5.0.htm

```diff
@@ -190,7 +190,7 @@ <h3>Major bugs fixed</h3>
 
 <h3>Backward incompatibities</h3>
 <ul>
-    <li>Classes that used<code>InterfilePDFSHeader</code> now contains a <code>shared_ptr&lt;ProjDataInfo&gt;</code> instead of a raw pointer, removing a memory leak.</li>
+    <li>Classes that use<code>InterfilePDFSHeader</code> now contain a <code>shared_ptr&lt;ProjDataInfo&gt;</code> instead of a raw pointer, removing a memory leak.</li>
```

Comment:
```suggestion
    <li>Classes that use <code>InterfilePDFSHeader</code> now contain a <code>shared_ptr&lt;ProjDataInfo&gt;</code> instead of a raw pointer, removing a memory leak.</li>
```